website/project/model.py

```diff
@@ -3381,7 +3381,8 @@ def require_approval(self, user, notify_initiator_on_complete=False):
     institution_domain = fields.StringField(list=True)
```

Comment:
As suggested in a previous comment, this should probably be changed to `institution_domains`